x-pack/plugins/alerting/server/rules_client/lib/siem_legacy_actions/migrate_legacy_actions.ts

```diff
@@ -66,7 +64,12 @@ export const migrateLegacyActions: MigrateLegacyActions = async (
         // set to undefined to avoid both per-actin and rule level values clashing
         throttle: undefined,
         notifyWhen: undefined,
-        actions: injectReferencesIntoActions(ruleId, legacyActions, legacyActionsReferences),
+        actions: transformRawActionsToDomainActions({
```

Comment:
All good since I can see we do `injectReferencesIntoActions` in `transformRawActionsToDomainActions`